hvpy/api_groups/movies/tests/test_get_movie_status.py

```diff
@@ -6,7 +6,7 @@
 
 def test_json_response():
     response = getMovieStatus(
-        id="VXvX5",
+        id="h2n6n",
```

Comment:
This is my custom movie made in the beta version of the API, the Id in the example throws an error. 
```
{
"error": "Unable to find the requested movie: 217592916",
"errno": 24
}
```

Reply:
Did you change the ID in the example?

Reply:
No, I did nothing, the example id doesn't work.

Reply:
Can it be changed to one that works?

Reply:
If not, it should be skipped.

Reply:
Okay.